optuna/study.py

```diff
@@ -880,6 +880,21 @@ def load_study(
     # type: (...) -> Study
     """Load the existing :class:`~optuna.study.Study` that has the specified name.
 
+    Example:
+
+        .. testcode::
+
+            import optuna
+
+            def objective(trial):
+                x = trial.suggest_float("x", 0, 10)
+                return x ** 2
+
+            study = optuna.create_study(storage="sqlite:///example.db", study_name="my_study")
+            study.optimize(objective, n_trials=3)
+
+            loaded_study = optuna.load_study(study_name="my_study", storage="sqlite:///example.db")
```

Comment:
Could you clean up the `example.db` at the end of the execution? It may affect the execution of other test cases that use `example.db`, and it would be safe if we remove it. I think we can use the [`testcleanup` directive](https://www.sphinx-doc.org/en/master/usage/extensions/doctest.html#directive-testcleanup) for this purpose.

Reply:
Not a must, but if possible, we could check for its existence in `.. testsetup::` and exit with a more graceful message.